client/styles/importer.less

```diff
@@ -1,3 +1,5 @@
+@cache-version: `process.env.npm_package_version` ;
```

Comment:
It's not a Node thing, it's an [npm thing](https://docs.npmjs.com/misc/scripts#packagejson-vars), and it only works when you run the app with `npm start` rather than something like `node app.js` (so it might fail in production depending on our config).

I don't think this is ideal, so I'll try to find a better way.